lib/index.js

```diff
@@ -34,10 +55,14 @@ function createPreprocessor(args, config, logger, helper) {
 }
 
 function createOptions(customConfig, baseConfig, helper, file) {
-  // Ignore 'base' property of custom preprocessor's config.
   customConfig = helper.merge({}, customConfig);
+
+  // Ignore 'base' property of custom preprocessor's config.
   delete customConfig.base;
 
+  // Ignore 'cachePath', which we use here instead of passing to babel.
+  delete baseConfig.cachePath;
```

Comment:
Please don't mutate objects that this function doesn't create. Could you copy it and remove the property like it was/is done for `base`? Also, could you also do the same for `customConfig`?

Maybe you could define a helper function to copy a given object and remove given properties.

Reply:
Will do.